### ISSUE TITLE
Remember last docked position on comfy restart

### ISSUE THREAD

**Issue by wogam** [hayden-fr/ComfyUI-Model-Manager#19]
Excellent add-on, but it should remember where it was last docked (to the left, right, etc) when comfy restarts/browser reloaded.

**Comment by cdb-boop** [comment]
Yeah, this shouldn't be hard to do. I'll get to it when I get some free time.
